### PR TITLE
#509 Enceladus DAO creds auth broken

### PR DESCRIPTION
**Pull request by lokm01**
- this was an error during PR refactoring...

### CODE REVIEW

File: dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala [hunk 1]
```diff
@@ -75,11 +76,15 @@ object EnceladusRestDAO extends EnceladusDAO {
     }
   }
 
-  private def restTemplateLogin(restTemplate: RestTemplate, url: String, username: String): Boolean = {
+  private def restTemplateLogin(restTemplate: RestTemplate, url: String, username: String, reqBody: Option[LinkedMultiValueMap[String, String]] = None): Boolean = {
```
Comment (Zejnilovic) [inline dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala]: How about 1 param per line?

File: dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala [hunk 2]
```diff
@@ -111,9 +116,13 @@ object EnceladusRestDAO extends EnceladusDAO {
   }
 
   def postLogin(username: String, password: String) = {
```
Comment (Zejnilovic) [inline dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala]: I know it is a pain, but could you annotate this method

**Comment by Zejnilovic** [comment]
Runs well

File: dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala (hunk 2 above)
Comment (lokm01) [reply dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala]: np

**Pull request merged**